src/library/basetracktablemodel.cpp

```diff
@@ -643,8 +643,15 @@ QVariant BaseTrackTableModel::roleValue(
             }
             QDateTime dt = mixxx::localDateTimeFromUtc(rawValue.toDateTime());
             if (role == Qt::ToolTipRole || role == kDataExportRole) {
+                // localized text date: "Wednesday, May 20, 1998 03:40:13 AM CEST"
                 return dt;
             }
+            if (field == ColumnCache::COLUMN_PLAYLISTTRACKSTABLE_DATETIMEADDED) {
+                // Timstamp column in history feature:
+                // Use localized date/time format without text: "5/20/98 03:40 AM"
+                return QLocale().toString(dt, QLocale::NarrowFormat);
```

Comment:
https://doc.qt.io/qt-6/qlocale.html#FormatType-enum
> Note: NarrowFormat might contain the same text for different months and days. It can even be an empty string if the locale doesn't support narrow names, so you should avoid using it for date formatting. Also, for the system locale this format is the same as ShortFormat.

We already have `displayLocalDateTime()` for this purpose.